ciao-controller/controller_test.go

```diff
@@ -823,7 +814,12 @@ func testStartWorkload(t *testing.T, num int, fail bool, reason payloads.StartFa
 }
 
 func testStartWorkloadLaunchCNCI(t *testing.T, num int) (*testutil.SsntpTestClient, []*types.Instance) {
-	netClient := newTestClient(0, ssntp.NETAGENT)
+	netClient, err := testutil.NewSsntpTestClientConnection("StartWorkloadLaunchCNCI", ssntp.NETAGENT)
+	if err != nil {
+		t.Fatal(err)
+	}
+	// deliberately leak
+	//defer netClient.Ssntp.Close()
 
```

Comment:
same as above, replace comment with "Caller is responsible for closing connection"